### ISSUE TITLE
test_jwt_login fails with vault 1.16

### ISSUE THREAD

**Issue by yan12125** [hvac/hvac#1152]
When I was looking into https://github.com/hvac/hvac/issues/1141, I noticed there is a new test failure, which happens after Arch Linux upgrades to vault 1.16.x:
```
_______________________ TestJWT.test_jwt_login_0_success _______________________

a = (<tests.integration_tests.api.auth_methods.test_jwt.TestJWT testMethod=test_jwt_login_0_success>,)

    @wraps(func)
    def standalone_func(*a):
>       return func(*(a + p.args), **p.kwargs)

/usr/lib/python3.11/site-packages/parameterized/parameterized.py:533:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tests/integration_tests/api/auth_methods/test_jwt.py:271: in test_jwt_login
    response = self.client.auth.jwt.configure(
hvac/api/auth_methods/jwt.py:113: in configure
    return self._adapter.post(
hvac/adapters.py:159: in post
    return self.request("post", url, **kwargs)
hvac/adapters.py:408: in request
    response = super().request(*args, **kwargs)
hvac/adapters.py:376: in request
    self._raise_for_error(method, url, response)
hvac/adapters.py:294: in _raise_for_error
    utils.raise_for_error(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

method = 'post', url = 'https://127.0.0.1:33021/v1/auth/jwt-test/config'
status_code = 400, message = None
errors = ['error checking jwks URL https://127.0.0.1:33021/v1/identity/oidc/.well-known/keys']
text = '{"errors":["error checking jwks URL https://127.0.0.1:33021/v1/identity/oidc/.well-known/keys"]}\n'
json = {'errors': ['error checking jwks URL https://127.0.0.1:33021/v1/identity/oidc/.well-known/keys']}

    def raise_for_error(
        method, url, status_code, message=None, errors=None, text=None, json=None
    ):
        """Helper method to raise exceptions based on the status code of a response received back from Vault.

        :param method: HTTP method of a request to Vault.
        :type method: str
        :param url: URL of the endpoint requested in Vault.
        :type url: str
        :param status_code: Status code received in a response from Vault.
        :type status_code: int
        :param message: Optional message to include in a resulting exception.
        :type message: str
        :param errors: Optional errors to include in a resulting exception.
        :type errors: list | str
        :param text: Optional text of the response.
        :type text: str
        :param json: Optional deserialized version of a JSON response (object)
        :type json: object

        :raises: hvac.exceptions.InvalidRequest | hvac.exceptions.Unauthorized | hvac.exceptions.Forbidden |
            hvac.exceptions.InvalidPath | hvac.exceptions.RateLimitExceeded | hvac.exceptions.InternalServerError |
            hvac.exceptions.VaultNotInitialized | hvac.exceptions.BadGateway | hvac.exceptions.VaultDown |
            hvac.exceptions.UnexpectedError

        """
>       raise exceptions.VaultError.from_status(
            status_code,
            message,
            errors=errors,
            method=method,
            url=url,
            text=text,
            json=json,
        )
E       hvac.exceptions.InvalidRequest: error checking jwks URL https://127.0.0.1:33021/v1/identity/oidc/.well-known/keys, on post https://127.0.0.1:33021/v1/auth/jwt-test/config

hvac/utils.py:41: InvalidRequest
```

Could you have a look?

**Comment by briantist** [comment]
Thanks @yan12125 this is a good callout, I've also opened this ticket to add 1.16 to CI so that we can catch things like this:
- https://github.com/hvac/hvac/issues/1154

**Comment by briantist** [comment]
Resolved by #1155
